server/src/test/java/io/crate/planner/operators/LogicalPlannerTest.java

```diff
@@ -429,14 +429,15 @@ public void test_limit_distinct_limits_outputs_to_the_group_keys_if_source_has_m
     @Test
     public void test_limit_on_join_is_rewritten_to_query_then_fetch() {
         LogicalPlan plan = plan("SELECT * FROM t1, t2 LIMIT 3");
-        assertThat(plan).isEqualTo(
-            """
-            Fetch[a, x, i, b, y, i]
-              └ Limit[3::bigint;0]
-                └ NestedLoopJoin[CROSS]
-                  ├ Collect[doc.t1 | [_fetchid] | true]
-                  └ Collect[doc.t2 | [_fetchid] | true]
-            """
+        assertThat(plan).hasOperators(
```

Comment:
Did this break the fetch optmization here, or is this ok ? Does the fetch Operator has to be on top of the Join ? @mfussenegger